src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+@font-face {
+  src: url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+}
```

Comment:
Use simple import or links in html head.
font-face is better used with local font files.